Makefile

```diff
@@ -186,12 +186,31 @@ run-test-multikueue-e2e-%: FORCE
 	@echo Running multikueue e2e for k8s ${K8S_VERSION}
 	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) ./hack/multikueue-e2e-test.sh
 
+.PHONY: install-shellcheck
+install-shellcheck: 
+	# removing previous shellcheck files if shellcheck give errors and warnings it wouldn't remove the downloaded files
+	rm -rf shellcheck-v0.9.0*
+	@echo "Installing ShellCheck"
+	curl -s https://api.github.com/repos/koalaman/shellcheck/releases/latest \
+    | grep "browser_download_url.*linux.x86_64.tar.xz" \
+    | cut -d : -f 2,3 \
+    | tr -d \" \
+    | wget -qi -
+	# extracting tar.gz file
+	tar -xvf shellcheck-v0.9.0.linux.x86_64.tar.xz
```

Comment:
Should we download the binary to `$PROJECT_DIR/bin` as well as other tools?

https://github.com/kubernetes-sigs/kueue/blob/4593b42139f5c75df36f16aee66e62dc405fa12d/Makefile#L297-L301

And then, should we call the `$PROJECT_DIR/bin/shellcheck` in the Make `shell-lint` target?